pinot-perf/src/main/java/org/apache/pinot/perf/BenchmarkRangeIndex.java

```diff
@@ -196,7 +196,11 @@ public void setup()
     @TearDown(Level.Trial)
     public void tearDown()
         throws IOException {
-      FileUtils.forceDelete(_indexDir);
+      try {
+        FileUtils.forceDelete(_indexDir);
```

Comment:
This was changed in the cleanup last time, but is actually necessary, depending on the OS the tmp file will get cleaned up aggressively so this may or may not throw and may or may not be necessary, but doing this and throwing fails the run.

Reply:
I think we did it in a lot of places. We might want to create a PR to change all of them to `FileUtils.deleteQuietly()`?

Reply:
Yes fine by me